hs_core/views/__init__.py

```diff
@@ -58,6 +61,24 @@ def verify(request, *args, **kwargs):
             u.is_active=True
             u.save()
             u.groups.add(Group.objects.get(name="Hydroshare Author"))
+
+        # register the user in HSAccess db accordingly
+        try:
+            ha_obj = HSAlib.HSAccess('admin', 'unused', django_settings.HS_ACCESS_DB, django_settings.HS_ACCESS_USERNAME,
+                                     django_settings.HS_ACCESS_PASSWORD, django_settings.HS_ACCESS_HOST, django_settings.HS_ACCESS_PORT)
+        except:
+            # unable to connect to the database
+            raise DatabaseError("unable to connect to the database HSAccess.")
+
+        try:
+            ha_obj.assert_user(str(u.username), 'HydroShare User')
+        except:
+            # unable to connect to the database
+            raise OperationalError("login %s cannot be inserted into HSAccess database." % u.username)
```

Comment:
there has to be a way to handle this more gracefully... and maybe that would solve our other problem of this verify view not being used any longer.

I wonder if it would be worth considering a sync task in Celery that could retry when fails, and basically try to add new new users to HSAccess periodically. That would get around both issues nicely.